components/sidebar-navigation/server.jsx

```diff
@@ -2,9 +2,9 @@ import renderComponent from "../../packages/utils/render-component";
 import Component from "./Component";
 
 export default async (args) => {
-  return renderComponent({
-    Component,
-    componentName: "sidebar-navigation",
-    args,
-  });
+  // return renderComponent({
```

Comment:
Since this is a client rendered component with SSJS this file is not needed. Hannah recommended leaving the file as it is based on other examples like `basic-story-hero`